### PR TITLE
[OCPBUGS-6849]: Update commands to find etcd pods for OCP 4.9

### PR DESCRIPTION
**Pull request by tmalove**
[OCPBUGS-6849](https://issues.redhat.com/browse/OCPBUGS-6849)
** Reviewers: This PR is to CP this update to the 4.9 branch. The original Jira applied to releases 4.9+, but this update was completed separately.

Version(s): 4.9

Links to docs preview:
- Restoring to a previous cluster stats (Steps 11b and 18): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp49-tlove/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state 
- Manual defragmentation (step 1a): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp49-tlove/scalability_and_performance/recommended-host-practices.html#manual-defrag-etcd-data_recommended-host-practices
- Determining the state of the unhealthy etcd member (step 3b): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp49-tlove/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-determine-state-etcd-member_replacing-unhealthy-etcd-member
- Replacing an unhealthy etcd member whose etcd pod is crashlooping (step 2a): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp49-tlove/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-crashlooping-etcd-member_replacing-unhealthy-etcd-member
- Replacing an unhealthy  bare metal etcd member whose machine is not running or whose node is not ready (step 1a): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp49-tlove/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-baremetal-etcd-member_replacing-unhealthy-etcd-member
- Replacing an unhealthy etcd member whose machine is not running or whose node is not ready (step 1a and verification, step 1): http://file.rdu.redhat.com/tlove/etcd-ocpbugs-6849-cp49-tlove/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member

QE review:
- [x ] QE has approved this change (Ge Liu).

### CODE REVIEW

**Comment by tmalove** [comment]
Hi @wking   Review request: this PR is for release 4.9. I made the changes directly in the release. Thanks!

File: modules/restore-replace-stopped-baremetal-etcd-member.adoc [hunk 1]
```diff
@@ -570,7 +570,7 @@ In a terminal that has access to the cluster as a `cluster-admin` user, run the
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-etcd -o wide | grep etcd | grep -v guard
+$ oc -n openshift-etcd get pods -l k8s-app=etcd
```
[the remark this answers is not in the document]
Comment (tmalove) [reply modules/restore-replace-stopped-baremetal-etcd-member.adoc]: @wking The command and link are updated for your review. Thanks.

**Review by wking — APPROVED** [review]
/lgtm

**Comment by tmalove** [comment]
Hi @geliu2016. Will you verify this update for 4.9? Thanks!

**Comment by geliu2016** [comment]
Verified it works well
/lgtm

**Review by opayne1 — COMMENTED** [review]
Your commits need squashing, but everything else looks good to me.

Note: Tami and I talked via slack why the changes are in 4.9 and not main. 

> The original PR in main, the CP failed for 4.9 and 4.10 and decided to make the changes directly in those 2 releases. Everything is merged for 4.11+. It is just 4.9 and 4.10 that I had to resolve.

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**